src/com/android/launcher3/BubbleTextView.java

```diff
@@ -392,8 +392,7 @@ protected void applyIconAndLabel(ItemInfoWithIcon info) {
         boolean useTheme = shouldUseTheme();
         FastBitmapDrawable iconDrawable = info.newIcon(getContext(), useTheme);
         mDotParams.appColor = iconDrawable.getIconColor();
-        mDotParams.color = getContext().getResources()
-                .getColor(android.R.color.system_accent3_200, getContext().getTheme());
+        mDotParams.color = iconDrawable.getIconColor();
```

Comment:
Lawnchair has a color theming system for these dynamic colors. Please add accent3_200 [here](https://github.com/LawnchairLauncher/lawnchair/blob/13-dev/lawnchair/src/app/lawnchair/theme/color/ColorTokens.kt) and use it instead.